src/plugins/saved_objects_management/public/management_section/objects_table/components/flyout.tsx

```diff
@@ -78,8 +79,10 @@ import { FailedImportConflict, RetryDecision } from '../../../lib/resolve_import
 import { OverwriteModal } from './overwrite_modal';
 import { ImportModeControl, ImportMode } from './import_mode_control';
 import { ImportSummary } from './import_summary';
+// eslint-disable-next-line @osd/eslint/no-restricted-paths
+import { DataSourcePicker } from '../../../../../data_source_management/public/components/data_source_picker/data_source_picker.js';
 
-const CREATE_NEW_COPIES_DEFAULT = false;
+const CREATE_NEW_COPIES_DEFAULT = true;
```

Comment:
I'm wondering if there any tests related to this in the [function test ](https://github.com/opensearch-project/opensearch-dashboards-functional-test)repo if we changed the default behavior

Reply:
The FT repo doesn't have tests for import saved objects, and the dashboard repo has functional tests which have been fixed to adopt the change of default behavior